npm/private/npm_translate_lock_state.bzl

```diff
@@ -140,6 +140,9 @@ def _init_common_labels(priv, rctx, label_store):
     # pnpm-workspace.yaml file
     label_store.add_sibling("lock", "pnpm_workspace", PNPM_WORKSPACE_FILENAME)
 
+    # yq is used for parsing the pnpm lock file (WORKSPACE only)
```

Comment:
why WORKSPACE only? That seems scary

Reply:
actually, that comment is out-dated. I figured out how to wire it up to in the module extension context as well now.